torch/quantization/fx/quantization_patterns.py

```diff
@@ -378,6 +378,27 @@ def convert(self, quantizer, node, load_arg, debug=False, convert_custom_config_
             load_arg(quantized=[0])(self.bn_node.args),
             load_arg(quantized=False)(self.bn_node.kwargs))
 
+@register_quant_pattern(torch.nn.Embedding)
+class Embedding(QuantizeHandler):
+    def __init__(self, quantizer, node):
+        super().__init__(quantizer, node)
+        assert node.op == 'call_module'
+        self.emb_node = node
+        self.emb = quantizer.modules[self.emb_node.target]
+
+    def convert(self, quantizer, node, load_arg, debug=False, convert_custom_config_dict=None):
+        qconfig = quantizer.qconfig_map[node.name]
+        assert not activation_is_statically_quantized(qconfig)
+        qemb = nnq.Embedding
+        quantized = qemb.from_float(self.emb)
+        parent_name, name = _parent_name(self.emb_node.target)
+        setattr(quantizer.modules[parent_name], name, quantized)
+        return quantizer.quantized_graph.create_node(
+            'call_module',
+            self.emb_node.target,
+            (load_arg(quantized=False)(self.emb_node.args[0]),), {})
```

Comment:
since we don't quantize anything I think it would be better to use
```
return quantizer.quantized_graph.create_node(
            'call_module',
            self.emb_node.target,
            load_arg(quantized=False)(self.emb_node.args), load_arg(quantized=False)(self.emb_node.kwargs))
```

and later we might be able to generalize this to other wight only quant ops.